### DISCUSSION TITLE
Update "Clear" button CSS styles for improved accessibility

### DISCUSSION THREAD

**Discussion by ZachSaucier** [Pagefind/pagefind#536] [Ideas]
The way that the "Clear" button currently functions, there's arguably two issues:

1. The click area to the left of the text is very small (2px).
2. When the button is navigated to via keyboard, the border makes the element look awkward:
    ![Screenshot 2023-12-26 at 6 37 32 PM](https://github.com/CloudCannon/pagefind/assets/3966722/fc1c56ad-ce55-460a-9075-2a169b8d235f)

Both of these issues would be fixed by changing the `padding` of `.pagefind-ui__search-clear` from `0 calc(15px * var(--pagefind-ui-scale)) 0 calc(2px * var(--pagefind-ui-scale))` to `0 calc(15px * var(--pagefind-ui-scale))`.

![Screenshot 2023-12-26 at 6 38 07 PM](https://github.com/CloudCannon/pagefind/assets/3966722/bd075b05-9bac-4048-893a-e08c7bd43edc)

Thanks!

**Comment by bglw** [comment]
Good notes, thanks! Converted to issue for dev.

**Reply by ZachSaucier** [reply]
Issue for reference: #537
